app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <string name="app_name">Data Collection</string>
     <string name="app_name_with_version">Data Collection %s</string>
-    <string name="app_version">0.1</string>
+    <string name="app_version">0.1.1</string>
```

Comment:
Interesting, we should resolve our differences here. I thought changing the version number was done within the app's [gradle](https://github.com/Esri/mapbook-android/blob/fb9319742dbc205873a3f22f6e47eccecbf1b5fc/mapbook-app/build.gradle#L10) file. @puneet-pdx can you weigh in here?

Reply:
@esreli I thought so too based on your other PR, but in the [`build.gradle`](https://github.com/Esri/data-collection-android/blob/master/app/build.gradle#L17) file, it's simply noted as `"1.0"` and doesn't look like it affects what's written out to the UI.

@puneet-pdx your insight here would be helpful.

Reply:
The app currently is not reading the VersionName from `build.gradle` file when we are displaying it in the UI. So Unfortunately we will have to do the duplication here. I can fix this in a separate PR.

Reply:
The versionName needs to be updated here as well https://github.com/Esri/data-collection-android/blob/master/app/build.gradle#L17